ts/packages/cli/src/commands/export.ts

```diff
@@ -13,6 +24,21 @@ export const CONFIG_EXPORT_RUN_DEFAULT_ERROR_TEXT = 'could not export configurat
 
 type TemplateContext = { [key: string]: string } | { key: string; value: string }[];
 
+const CASE_FUNCTION: { [casing: string]: (str: string) => string } = {
```

Comment:
`CASE_FUNCTION` -> `casingFormatters`, this change will align with the existing `configFormatters` from `lib` in terms of the naming pattern.

Reply:
It is a const, just for the sake of clarity:
![image](https://github.com/configu/configu/assets/47181182/6389e7a4-d3f0-4b28-8097-ff3331288ab8)